src/components/inline-notification/daikin-inline-notification.ts

```diff
@@ -0,0 +1,123 @@
+import { LitElement, css, html, nothing, unsafeCSS } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import tailwindStyles from "../../tailwind.css?inline";
+import {
+  cvaContainer,
+  cvaContent,
+  cvaTimestamp,
+  formatDate,
+} from "../../utils/notification-common";
+import type { ToastNotificationVariantProps } from "../toast-notification";
+
+/**
+ * The inline notification component is a UI element used to inform users about important updates, alerts, or messages within an application.
+ *
+ * Alerts appear within the content of the application, usually embedded directly within a page or section.
+ * Alerts are more persistent and are used to highlight important information or status updates that should remain visible to the user until they are acknowledged or the issue is resolved.
+ *
+ * @fires close - A custom event emitted when a user clicks the close button.
+ *
+ * @slot title - A slot for the inline-notification title content.
+ * @slot description - A slot for the inline-notification description content.
+ *
+ * @example
+ *
+ * ```js
+ * import "@daikin-oss/design-system-web-components/components/inline-notification/index.js";
+ * ```
+ *
+ * ```html
+ * <daikin-inline-notification>
+ *   <span slot="title">InlineNotification title</span>
+ *   <span slot="description">InlineNotification description</span>
+ * </daikin-inline-notification>
+ * ```
+ */
+@customElement("daikin-inline-notification")
+export class DaikinInlineNotification extends LitElement {
+  static override readonly styles = css`
+    ${unsafeCSS(tailwindStyles)}
+
+    :host {
+      display: block;
+      width: 100%;
+    }
+
+    :host([hidden]) {
+      display: none;
+    }
+  `;
+
+  /**
+   * Status of the notification.
+   */
+  @property({ type: String })
+  status: ToastNotificationVariantProps["status"] = "positive";
+
+  /**
+   * Specifies how to arrange the elements.
+   */
+  @property({ type: String })
+  layout: ToastNotificationVariantProps["layout"] = "horizontal";
+
+  /**
+   * Whether to display the close button.
+   */
+  @property({ type: Boolean, reflect: true })
+  closable = false;
+
+  /**
+   * Whether to display the timestamp.
+   */
+  @property({ type: Boolean, reflect: true })
+  timestamp = false;
```

Comment:
The timestamp does not necessarily correspond to the date and time of creation of the element.

```suggestion
  /**
   * The timestamp to display.
   */
  @property({ type: Object, reflect: true, attribute: false })
  timestamp: Date | null = null;
```